mmv1/products/netapp/storagePool.yaml

```diff
@@ -25,9 +25,21 @@ description: |
  The capacity of the pool can be split up and assigned to volumes within the pool. Storage pools are a billable
  component of NetApp Volumes. Billing is based on the location, service level, and capacity allocated to a pool
  independent of consumption at the volume level.
+
+ Storage pools of service level Flex are available as zonal (single zone) or regional (two zones in same region) pools.
```

Comment:
Nit: we may want to note some or all of this is beta-only

Reply:
done, added a note